app/assets/javascripts/discourse/app/initializers/subscribe-user-changes.js

```diff
@@ -1,15 +0,0 @@
-export default {
-  name: "subscribe-user-changes",
```

Comment:
Remove an extra initializer when we already have one that is used for user specific MessageBus subscriptions.